heudiconv/dicoms.py

```diff
@@ -109,6 +109,11 @@ def create_seqinfo(mw: dw.Wrapper, series_files: list[str], series_id: str) -> S
         date=dcminfo.get("AcquisitionDate"),
         series_uid=dcminfo.get("SeriesInstanceUID"),
         time=dcminfo.get("AcquisitionTime"),
+        # ASL support
+        in_plane_phase_encoding_direction = dcminfo.get("InPlanePhaseEncodingDirection") if dcminfo.get("InPlanePhaseEncodingDirection") else None,
```

Comment:
just
```suggestion
        in_plane_phase_encoding_direction = dcminfo.get("InPlanePhaseEncodingDirection"),
```
and so on would be sufficient.  But first please elaborate why these particular are necessary for ASL support?
Note: we do not really want to extend list of extracted fields to get unnecessarily long.

NB I should really look into finishing https://github.com/nipy/heudiconv/pull/581 some days soon to provide means for easy extraction of any metadata desired.